common/app/common/InlineStyles.scala

```diff
@@ -93,21 +94,48 @@ object InlineStyles {
   }
 
   /**
+    * Execute some code n times, or until no exception is thrown
```

Comment:
I think that since this is fairly generic retry logic, it probably belongs outside of InlineStyles